src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+
+    public Colors getRandomColor() {
+        return Colors.values()[random.nextInt(Colors.values().length)];
+    }
+
+    public Random getRandom() {
+        return random;
```

Comment:
The method 'getRandom()' is unnecessary if it's not used elsewhere in your code. Consider removing it to keep your code clean and focused on the task requirements.